src/renderer/store/actions/LibraryActions.ts

```diff
@@ -332,6 +332,22 @@ export const incrementPlayCount = async (source: string): Promise<void> => {
   }
 };
 
+/**
+ * Update the id3 attributes.
+ */
+export const updateTrack = async (track: Track): Promise<void> => {
+  const query = { path: track.path }; // HACK Not great, should be done with an _id
+  const update = track;
+  // const update = { $set: { title: track.title, artist: [track.artist], album: track.album } };
+  try {
+    await app.db.Track.updateAsync(query, update);
```

Comment:
You may need to update the `loweredMetas`, otherwise search will probably not work unless people reset and rescan their libraries